src/applications/accredited-representative-portal/containers/POARequestIndividualSearchPage.jsx

```diff
@@ -0,0 +1,228 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  VaTextInput,
+  VaDate,
+  VaLoadingIndicator,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import SsnField from 'platform/forms-system/src/js/web-component-fields/SsnField';
+import { useSearchParams, useNavigation, Form } from 'react-router-dom';
```

Comment:
One note @pixiitech. Is it the case that we aren't using anything from `react-router`'s `Form`? Will this work with `react`'s `form` instead? If so, by changing to `form`, there might be situations where we make the client download less javascript.